### PR TITLE
Added tlg0081.tlg007.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Dionysius Halicarnassensis, Libri secundi de antiquis oratoribus reliquiae

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +59 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +22389 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0081/__cts__.xml`                          | New      |
| `./data/tlg0081/tlg007/__cts__.xml`                   | New      |
| `./data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml` | New      |
| `./data/tlg0081/__cts__.xml`                          | Passing  |
| `./data/tlg0081/tlg007/__cts__.xml`                   | Passing  |
| `./data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3540)*

**Comment by AlisonBabeu** [comment]
Since I'm being snoopy and looking at pull requests before they are even accepted, you need to change the editor in the edition level cts_xml file, its not Jahn, its Usener and Radermacher, which you have correctly in the TEI-XML file! Cheers!

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +59 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +22389 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0081/__cts__.xml`                          | New      |
| `./data/tlg0081/tlg007/__cts__.xml`                   | New      |
| `./data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml` | New      |
| `./data/tlg0081/__cts__.xml`                          | Passing  |
| `./data/tlg0081/tlg007/__cts__.xml`                   | Passing  |
| `./data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3542)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +59 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +22389 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0081/__cts__.xml`                          | New      |
| `./data/tlg0081/tlg007/__cts__.xml`                   | New      |
| `./data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml` | New      |
| `./data/tlg0081/__cts__.xml`                          | Passing  |
| `./data/tlg0081/tlg007/__cts__.xml`                   | Passing  |
| `./data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3548)*

File: data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,4807 @@
...
+παραδειγμάτων.</p>
+</div>
+
+<div type="textpart" subtype="chapter" n="14">
+<p>14 ὥστε περὶ μὲν τοῦ χαρακτῆρος . τοῦ δὲ
+ῥητορικοῦ γένους τοῦ μεταξύ τῶν ἄκρων ἑκατέρων, ὃ
+<lb n="15"/> ἀτελὲς παραλαβὼν ὁ Δημοσθένης παρʼ Ἰσοκράτους τε
+καὶ ἔτι προτέρου Θρασυμάχου καὶ τελευταίου Πλάτωνος <milestone unit="996"/>
+ἐτελείωσεν ὅσον ἦν ἀνθρωπίνη φύσει δυνατόν,
+πολλὰ μ
```
Comment (planatheisa) [inline data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml]: There are section-divs in this chapter (and only in this one) but the CTS only has one level. Shouldn't the sections appear there too?

**Comment by capitains-hook** [comment]
## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |       -6 |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3605)*

**Pull request merged**